ui/src/version/VersionEndpointActions.js

```diff
@@ -166,7 +171,6 @@ const VersionEndpointActions = ({
 
     // Undeploy
     if (
-      versionEndpoint.status === "pending" ||
```

Comment:
Thanks for this @ariefrahmansyah !

Reply:
I was thinking of replacing `actions.undeploy` with something like this instead but I guess this isn't too important:
```js
    undeploy: (undeployable) =>
      <EuiFlexItem grow={false} key={`undeploy-${versionEndpoint.id}`}>
        <EuiButtonEmpty
          onClick={openModal(props.toggleUndeployEndpointModal)}
          color="danger"
          iconType="exportAction"
          size="xs"
          disabled={undeployable}
        >
          <EuiText size="xs">Undeploy</EuiText>
        </EuiButtonEmpty>
      </EuiFlexItem>
    ,
```